deploy/bz1997062-crio-pid-leak/00-osd-cleanup-pids.DaemonSet.yaml

```
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: execpid-cleaner
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      name: execpid-cleaner
+  template:
+    metadata:
+      labels:
+        name: execpid-cleaner
+    spec:
+      tolerations:
```

This might also need tolerations for infra node taints or else it won't schedule to them.

```suggestion
      tolerations:
      - effect: NoSchedule
        key: node-role.kubernetes.io/infra
        operator: Exists      
```